shopfloor_reception/services/reception.py

```diff
@@ -999,18 +994,21 @@ def done_action(self, picking_id, confirmation=False):
             return self._response_for_confirm_done(
                 picking, message=self.msg_store.need_confirmation()
             )
-        self._handle_backorder(picking)
+        self._handle_backorder(picking, True)
```

Comment:
We not want to cancel the Backorder Always please Pass the cancel_backorder var. 
```suggestion
        self._handle_backorder(picking, cancel_backorder)
```